app/Filament/Resources/UserResource/Pages/ListUsers.php

```diff
@@ -16,4 +20,28 @@ protected function getHeaderActions(): array
             Actions\CreateAction::make(),
         ];
     }
+
+    /**
+     * @return array|Tab[]
+     */
+    public function getTabs(): array
+    {
+        $tabs = ['All Users' => Tab::make('All Users')->badge($this->getModel()::count())];
```

Comment:
@kolanaveen Here, please add condition for roles so that the query should be
```
$this->getModel()::whereHas('roles', function ($innerQuery) {
            return $innerQuery->where('name', '!=', User::ROLE_ADMIN);
 })->count()
```

Reply:
There is already a condition in `resource` so we need to use any one condition